libs/agno/agno/tools/mysql.py

```diff
@@ -0,0 +1,169 @@
+from typing import Any, Dict, Optional
+import mysql.connector
+from mysql.connector import Error
+from urllib.parse import urlparse
+from agno.tools import Toolkit
+from agno.utils.log import logger
+
+class MySQLTools(Toolkit):
+    """A basic tool to connect to a MySQL database and perform read-only operations on it."""
```

Comment:
We already have a SQLTools that uses sql alchemy. What benefit does the mysql connector provide above that?

Reply:
yes, closed :)